WordPress/gutenberg
# Data Module: Refactor the Post Type Capabilities fetching to use the data module

This pull request is part of a series of pull request targeted at more complex use of selector resolvers introduced in #5219.

The current PR moves the fetching of the post type user capabilities to the data module.

**Testing instructions**

Check that you can publish properly and that the sidebar panels show up as expected.
